src/Components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,143 @@
+@mixin flexDisplay($position, $direction) {
+  display: flex;
+  flex-direction: $direction;
+  align-items: center;
+  justify-content: $position;
+}
+
+$GDSC-blue: #547dbf;
+$White: #fff;
+$Shadow-color: rgb(218, 215, 215);
```

Comment:
Checkout `index.scss` if these colours are available or if not available then create a variable there (following the proper naming convention, check other colour variable names) and then use that variable in your Navbar using `var(variable-name)`